### ISSUE TITLE
mms/mmk differences

### ISSUE THREAD

**Issue by hb--** [endlesssoftware/mmk#84]
It seems .error is a MMK feature, which was/is not available in MMS. I stumbled over it in SQLITE3, when trying to build that with MMS:

.IF "$(MMSARCH_NAME)" .EQ "Alpha"
ARCH = AXP
.ELSIF "$(MMSARCH_NAME)" .EQ "IA64"
ARCH = I64
.ELSIF "$(MMSARCH_NAME)" .EQ "VAX"
ARCH = VAX
.ELSE
.ERROR You must define the ARCH macro as one of: VAX, AXP or I64
.ENDIF
(oh, shouldn't the error message say IA64?)

Anyway, it seems reasonable to mention this difference, as well as (OK, I didn't check the docs for this one):

$ mms /desc=tt: 
CFLAGS = $(CFLAGS)/NAME=AS_IS
CFLAGS = $(CFLAGS)/WARN=DISABLE=(LONGEXTERN,EMPTYFILE)
%MMS-W-MBREDEFILL, Illegal attempt to redefine macro CFLAGS
all :
        write sys$output "$(CFLAGS)"
 Exit 
write sys$output "/NOLIST/OBJECT=ALL.OBJ/NAME=AS_IS"
/NOLIST/OBJECT=ALL.OBJ/NAME=AS_IS
$

**Comment by tesneddon** [comment]
Yes, ```.ERROR``` is a unique feature of MMK.  I have no immediate recollection of the manual. I'll have a look and see if it needs updating. (Just looked, there are no directives documented in the manual).

You are also correct about the error message typo too :-)

As for the difference...well, that is because there is no ```/EXTENDED_SYNTAX``` qualifier that has been added to the ```MMS``` command.  I don't think that is really worth mention though as just about every useful feature of MMS requires this qualifier. Without there are no built-in functions, no redefinition of macros, etc.
